.circleci/config.yml

```diff
@@ -35,15 +45,57 @@ jobs:
       image: ubuntu-1604:201903-01
 
     environment:
+      GOVERSION: 1.12.17
       DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.13-base
       REPO_PATH: github.com/prometheus/node_exporter
+      OSXCROSS_PATH: /usr/osxcross
+      OSXCROSS_REV: 3034f7149716d815bc473d0a7b35d17e4cf175aa
+      SDK_VERSION: 10.11
+      DARWIN_VERSION: 15
+      OSX_VERSION_MIN: 10.6
+      OSXCROSS_SDK_URL: https://www.dropbox.com/s/bwlbtwywvlu1au7/MacOSX10.11.sdk.tar.xz
 
     steps:
     - checkout
     - run: docker run --privileged linuxkit/binfmt:v0.7
-    - run: make promu
-    - run: promu crossbuild
-    - run: promu --config .promu-cgo.yml crossbuild
+    - run:
+        name: install go
+        command: |
+          sudo rm -rf /usr/local/go
+          wget https://dl.google.com/go/go${GOVERSION}.linux-amd64.tar.gz
+          sudo tar -C /usr/local -xzf go${GOVERSION}.linux-amd64.tar.gz
```

Comment:
Hint: https://www.howtogeek.com/howto/uncategorized/linux-quicktip-downloading-and-un-tarring-in-one-step/

Reply:
👍